src/control/server/faults.go

```diff
@@ -65,6 +67,26 @@ var (
 	)
 )
 
+func FaultPoolNvmeTooSmall(reqBytes uint64, targetCount int) *fault.Fault {
+	return serverFault(
+		code.ServerPoolNvmeTooSmall,
+		fmt.Sprintf("requested NVMe capacity (%s) is too small",
+			humanize.Bytes(reqBytes)),
+		fmt.Sprintf("NVMe capacity should be larger than %s * number of targets (%d)",
+			humanize.Bytes(minNvmeBytes), targetCount),
+	)
+}
+
+func FaultPoolScmTooSmall(reqBytes uint64, targetCount int) *fault.Fault {
+	return serverFault(
+		code.ServerPoolScmTooSmall,
+		fmt.Sprintf("requested SCM capacity (%s) is too small",
+			humanize.IBytes(reqBytes)),
+		fmt.Sprintf("SCM capacity should be larger than %s * number of targets (%d)",
+			humanize.IBytes(minScmBytes), targetCount),
```

Comment:
I was initially going to object to displaying IEC units here, as it's inconsistent with the output of storage scan/pool info, etc. But I guess given that the minimum sizes are defined in these units it makes sense. Same for the NVMe size.

Reply:
that's why I specified as IEC , will do the same for NVMe